nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/ui/provenance-event-listing/provenance-event-table/lineage/lineage.component.ts

```diff
@@ -46,11 +46,13 @@ export class LineageComponent implements OnInit {
             this.addLineage(lineage.results.nodes, lineage.results.links);
         }
     }
+
     @Input() eventId: string | null = null;
+
     @Input() set eventTimestampThreshold(eventTimestampThreshold: number) {
         if (this.previousEventTimestampThreshold >= 0) {
-            let nodes: any = this.lineageContainerElement.selectAll('g.node');
-            let links: any = this.lineageContainerElement.selectAll('path.link');
+            let nodes: any = this.lineageContainerElement.selectAll('g.node.rendered');
+            let links: any = this.lineageContainerElement.selectAll('path.link.rendered');
```

Comment:
when showing/hiding these items we should also turn on/of pointer-events. Otherwise, you can still get to the context menus for the hidden items if you mouse over them